.github/workflows/integration-tests-azure.yml

```diff
@@ -11,11 +11,12 @@ jobs:
     name: Regular
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
-        python_version: ["3.8", "3.9", "3.10", "3.11"]
         profile: ["ci_azure_auto"]
+        python_version: ["3.11"]
```

Comment:
Is the adapter only compatible with py311? Or just to speed up tests, only run on 3.11?